cli_tools/diagnostics/main_windows.go

```diff
@@ -204,6 +208,30 @@ func collectFilePaths(roots []string) ([]string, []error) {
 	return filePaths, errs
 }
 
+// getPlainEventLogs generates plain text event logs thru `Get-WinEvent` powershell cmd,
+// return list file paths and errors(if any).
+func getPlainEventLogs(evts []winEvt, errs chan error) []string {
+	pwshPath, err := exec.LookPath("powershell")
+	if err != nil {
+		errs <- err
+		return []string{}
+	}
+	argStr := `Get-WinEvent %s'%s' | Format-Table -Auto -Wrap`
+	commands := make([]runner, 0)
+	for _, evt := range evts {
+		source := "-LogName "
+		if evt.fromProvider {
+			source = "-ProviderName "
+		}
+		commands = append(commands, cmd{
+			path:            pwshPath,
+			args:            fmt.Sprintf(argStr, source, evt.logName),
```

Comment:
The use of argStr is pretty far from its declaration here, making it a bit difficult to understand what's being executed. You could consider writing a small function

`func getCommandToConvertEvt(fromProvider bool) string { ... }`

then this loop would just iterate over `commands = append(... args: getCommandToConvertEvt(evt.fromProvider) ...)`